src/tests/conformance/cmd/validators.rs

```diff
@@ -67,9 +52,5 @@ async fn c015_TM_VALIDATOR_LIST_COLLECTION_node_should_send_validator_list() {
         }
         false
     };
-    assert!(synth_node.expect_message(&check).await);
-
-    // Shutdown.
-    synth_node.shut_down().await;
-    node.stop().expect("unable to stop the rippled node");
+    perform_response_test(Default::default(), &check).await;
```

Comment:
I'm wondering if there's a better name for this function that would not suggest that it has to be a response (as opposed to an "unsolicited" message like in this case) :thinking:

Reply:
Perhaps something like `perform_expected_message_test`?

Reply:
yeah, I think that's better; in any case, it can happen in a follow-up

Reply:
Done

Reply:
`expect_message` 
or if more info
`expect_message_post_handshake` 

In any case, it would be nice to have documentation for this function in mod.rs

Reply:
I've added some comments for this function scenario.